app/assets/javascripts/admin/team/team_list_view.js

```diff
@@ -95,6 +95,16 @@ class TeamListView extends React.PureComponent<Props, State> {
     });
   };
 
+  renderPlaceholder() {
+    return (
+      <React.Fragment>
+        {"There are no teams. You can "}
+        <a onClick={() => this.setState({ isTeamCreationModalVisible: true })}>add a team</a>
+        {" which can be used to allow or disallow access to specific datasets."}
```

Comment:
This sounds as if this was the only purpose of teams. Maybe `You can create teams to control access to specific datasets and manage which users can be assigned to a task type`?